pep/app/pods/most-cited/nav/template.hbs

```diff
@@ -1,113 +1,68 @@
-                                <ButtonSpinner
-                                    @type="primary"
-                                    @blockLayout={{false}}
-                                    @wide={{true}}
-                                    @showSuccess={{true}}
-                                    @label={{t "common.filter"}}
-                                    @action={{this.filterTableResults}}
-                                />
-                            </div>
-                        </form>
-                        <Alert
-                            @isShown={{this.hasTooManyResults}}
-                            @type="warning"
-                            @scrollableNamespace="sidebar-left"
-                            class="mb-0 border-left-0 rounded-0 small"
+                    <div class="form-group">
+                        <FlInput
+                            @placeholder={{t "mostCited.author"}}
+                            @value={{this.author}}
+                            maxlength="255"
+                            autocomplete="off"
+                        />
+                    </div>
+                    <div class="form-group">
+                        <FlInput
+                            @placeholder={{t "mostCited.title"}}
+                            @value={{this.title}}
+                            maxlength="255"
+                            autocomplete="off"
+                        />
+                    </div>
+                    <div class="form-group">
+                        <FlInput
+                            @value={{this.journal}}
+                            @placeholder={{t "mostCited.journal"}}
                         >
-                            <p class="mb-1 d-flex">
-                                <span class="mr-1">
-                                    <FaIcon @icon="info-circle" @size="lg" />
-                                </span>
-                                <strong>
-                                    {{t "search.form.tooManyResults.heading"}}
-                                </strong>
-                            </p>
-                            <p class="mb-1">
-                                {{t
-                                    "search.form.tooManyResults.numResults"
-                                    count=(format-number @resultsCount)
-                                    htmlSafe=true
-                                }}
-                            </p>
-                            <p class="mb-0">
-                                {{t "search.form.tooManyResults.instructions"}}
-                            </p>
-                        </Alert>
+                            <PowerSelect::Journal
+                                @selected={{this.journal}}
+                                @onChange={{this.updateJournal}}
+                            />
```

Comment:
we need to look at cleaning up the styling of the power selects to match the styles of the other inputs/native selects, but we can do that later.

Reply:
Thats fine - though I don't necessarily agree that they should look exactly like native selects because they are so different.